tutorial/output/ex00_ngsp_expected.xml

```diff
@@ -0,0 +1,313 @@
+<!--
+
+    pipeline  Consensus assembly and allele interpretation pipeline.
```

Comment:
sorry to be nit-picky here, but the pattern should be
```
    ${project-name}  ${project description}
    Copyright (c) ${copyright years} National Marrow Donor Program (NMDP)
```

So the header here and above should be something like
```
    flow-validation  Validation workflow.
    Copyright (c) 2015 National Marrow Donor Program (NMDP)
```
or of course if you can think of a better description that can be changed here and in the repository description.

This link may help
http://www.gnu.org/licenses/gpl-howto.en.html